utils/utils.jsx

```diff
@@ -1155,16 +1155,17 @@ export function offsetTopLeft(el) {
     return {top: rect.top + scrollTop, left: rect.left + scrollLeft};
 }
 
-export function getSuggestionBoxAlgn(textArea, pxToSubstract = 0, boxLocation = 'top') {
+export function getSuggestionBoxAlgn(textArea, pxToSubstract = 0/*, boxLocation = 'top'*/) {
```

Comment:
Any reason to keep the commented code?

Reply:
Good catch. That should've been removed.